hyp3_sdk/asf_search.py

```diff
@@ -48,15 +47,38 @@ def get_nearest_neighbors(granule: str, max_neighbors: int = 2,) -> List[dict]:
     Returns:
         neighbors: a list of neighbors sorted by time
     """
+    params = {
+        'output': 'json',  # jsonlite doesn't include centerLat/centerLon
+        'platform': 'S1',
+        'granule_list': granule,
+    }
+    response = requests.post(_SEARCH_API, params=params)
+    raise_for_search_status(response)
+    if not response.json()[0]:
+        raise ASFSearchError(f'Reference Sentinel-1 granule {granule} could not be found')
+    reference = [r for r in response.json()[0] if not r['processingLevel'].startswith('METADATA_')][0]
```

Comment:
Still a chance for an `IndexError` here if the search returns only `METADATA_` entries